### PR TITLE
AMB-362 capitalise endpoints in spec

### PR DESCRIPTION
**Pull request by RyanThomas1214**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/reasonable-adjustment-flag.yaml [hunk 1]
```diff
@@ -1421,7 +1421,7 @@ paths:
...
       tags:
         - Endpoints
-  /removerarecord:
+  /%removerarecord:
```
Comment (JalfResi) [inline specification/reasonable-adjustment-flag.yaml:1424]: Shouldnt this be a dollar sign?

Comment (RyanThomas1214) [reply specification/reasonable-adjustment-flag.yaml:1424]: Haha yep, I need glasses

**Review by JalfResi — APPROVED** [review]

File: specification/reasonable-adjustment-flag.yaml [hunk 2]
```diff
@@ -1063,7 +1063,7 @@ paths:
           content:
             application/fhir+json:
               schema:
-                $ref: '#/paths/~1flag/get/responses/400/content/application~1fhir%2Bjson/schema'
+                $ref: '#/paths/~1Flag/get/responses/400/content/application~1fhir%2Bjson/schema'
```
Comment (JalfResi) [inline specification/reasonable-adjustment-flag.yaml:1066]: ~1Flag?

File: specification/reasonable-adjustment-flag.yaml [hunk 3]
```diff
@@ -1275,7 +1275,7 @@ paths:
...
```
Comment (JalfResi) [inline specification/reasonable-adjustment-flag.yaml:1278]: ~1Flag?

File: specification/reasonable-adjustment-flag.yaml [hunk 4]
```diff
@@ -1329,7 +1329,7 @@ paths:
...
```
Comment (JalfResi) [inline specification/reasonable-adjustment-flag.yaml:1332]: ~1Flag??

**Review by JalfResi — APPROVED** [review]

**Pull request merged**